dolphin/drivers/fake_storage/__init__.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2020 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from dolphin.drivers import driver
+
+
+class FakeStorageDriver(driver.StorageDriver):
+    """FakeStorageDriver shows how to implement the StorageDriver,
+    it also plays a role as faker to fake data for being tested by clients"""
```

Comment:
According to the coding standards, the tail """ should be put at a new line

Reply:
thank you, done.